tests/test_path_follow_3d.h

```diff
@@ -39,25 +39,26 @@
 namespace TestPathFollow3D {
 
 TEST_CASE("[PathFollow3D] Sampling with unit offset") {
-	const Ref<Curve3D> &curve = memnew(Curve3D());
+	Ref<Curve3D> curve = memnew(Curve3D);
 	curve->add_point(Vector3(0, 0, 0));
 	curve->add_point(Vector3(100, 0, 0));
 	curve->add_point(Vector3(100, 100, 0));
 	curve->add_point(Vector3(100, 100, 100));
 	curve->add_point(Vector3(100, 0, 100));
-	const Path3D *path = memnew(Path3D);
+	Path3D *path = memnew(Path3D);
 	path->set_curve(curve);
-	const PathFollow3D *path_follow_3d = memnew(PathFollow3D);
+	PathFollow3D *path_follow_3d = memnew(PathFollow3D);
 	path->add_child(path_follow_3d);
+	path_follow_3d->notification(Node::NOTIFICATION_ENTER_TREE);
```

Comment:
`PathFollow3D` sets its `path` property to its parent `Path3D` on entering the tree but the `path` parent is not in the tree so the notification `NOTIFICATION_ENTER_TREE` is never emitted, I'm calling this method manually on every test. Is this appropriate for a test or too hacky?